plugins/repository-azure/src/main/plugin-metadata/plugin-security.policy

```diff
@@ -20,4 +20,5 @@
 grant {
   // azure client opens socket connections for to access repository
   permission java.net.SocketPermission "*", "connect";
+  permission java.lang.RuntimePermission "setFactory";
```

Comment:
From #53903 this will go away once #53903 is merged, I just set it to be able to run 3rd party tests locally and verify this actually works now.